### DISCUSSION TITLE
Sidewalk separate on one side only

### DISCUSSION THREAD

**Discussion by Helium314** [streetcomplete/StreetComplete#2635] [Q&A] [answered]
Sometimes there are sidewalks on both sides on the street, and there is a separate sidewalk on one side, but not on the other.
What would be the correct anwer for the sidewalk quest?
I can't answer something like _left: yes, right: displayed separately_

In case it matters / requires a different answer: I found this for sidewalks directly at the street (only separated by curb, and maybe parked cars) and for sidewalks that are separated from the street by trees or grass.

**Comment by matkoniecz** [comment, marked as answer]
In such case correct answer is "displayed separately" that will tag `sidewalk=separate` and does not contain information about where sidewalk is - that is provided by explicitly mapped sidewalk geometry.

**Reply by Helium314** [reply]
sorry, I see my question is missing a key info: what if there are sidewalks on both sides, but only one is displayed separate?

**Reply by matkoniecz** [reply]
I would create note that sidewalk is missing.

Or map missing sidewalk way directly in Vespucci.

And later answer that sidewalk is separately mapped, once it is mapped.

(this assumes that mapping as separate ways is standard in a given area, if `sidewalk` tags are standard and then proper fix may be to delete this way)

**Reply by andrewharvey** [reply]
> (this assumes that mapping as separate ways is standard in a given area, if sidewalk tags are standard and then proper fix may be to delete this way)

Sorry getting off topic, but I think that's a step too far. I map both sidewalk=left/right/both AND as a separate way, better for data consumers they can choose which to use. Not saying you need to do that, but if someone has traced out the sidewalk way you shouldn't delete their work unless it's not actually there.

**Reply by matkoniecz** [reply]
Note that in some areas the standard is to not map sidewalks as a separate way, in such case I would respect local communities and their decision to avoid separately mapped ways.

> I map both sidewalk=left/right/both AND as a separate way

I am not convinced that it is really OK, it seems to me that `sidewalk=separate` is preferable in such case - similarly `cycleway=track` on road + `highway=cycleway` on separate way seems incorrect to me. But it probably should be discussed elsewhere.

**Reply by mnalis** [reply]
I agree that leaving note is best to do there (preferably with pictures), and later in full-blown editor add missing way (or let others handle it). I am against removing way that someone mapped separately, as that is more precise, can be split differently, can contain extra tags, etc.
